scRNA-seq-advanced/04-differential_expression.Rmd

```diff
@@ -12,156 +12,260 @@ output:
+These 10 samples represent two different types of rhabdomyosarcoma (RMS): embryonal rhabdomyosarcoma (ERMS) and alveolar rhabdomyosarcoma (ARMS).
+These two subtypes can be distinguished by the presence of the `PAX3/PAX7-FOXO1` fusion gene present only in ARMS patients.
+Additionally, cells found in ARMS tumors tend to have an increased mutational burden and cells are in a more differentiated state in comparison to ERMS tumor cells ([Shern _et al._ 2014](https://doi.org/10.1158/2159-8290.CD-13-0639), [Stewart _et al._](https://doi.org/10.1016/j.ccell.2018.07.012)).
+RMS tumors, regardless of subtype, are made up of cells typically associated with development of skeletal muscle: mesoderm, myoblasts, and myocytes ([Sebire and Malone 2003](https://doi.org/10.1136/jcp.56.6.412)).
+The group that published the data we are going to look at in this notebook identified distinct differences between ARMS and ERMS tumors. 
+Here we will look at a subset of the samples they sequenced and walk through testing one of their hypothesis that cell types have distinct gene expression patterns in ARMS vs. ERMS samples.
 
 ## Set Up
 
-```{r}
+```{r setup}
+# set seed for reproducibility
 set.seed(2022)
 
-library(magrittr)
-library(SingleCellExperiment)
-library(ggplot2)
+# load libraries
+library(magrittr) # the pipe (%>%)
+library(ggplot2) # plotting functions
+library(SingleCellExperiment) 
 
+# package used for differential expression analysis
 library(DESeq2)
 ```
 
-```{r}
+We will start by reading in a `SingleCellExperiment` (SCE) object that contains both the uncorrected (merged but not integrated) and corrected (integrated) gene expression data for all 10 samples.
+
+Prior to integration all 10 samples went through the same filtering, normalization, and dimensionality reduction.
+These 10 samples were then merged into one `SingleCellExperiment` object following the same steps outlined in `03-dataset_integration.Rmd`.
+The merged object was then integrated with `fastMNN` to obtain a corrected gene expression assay and corrected reduced dimensionality results.
+The final SCE object was stored in `data/rms/integrated/rms_all_sce.rds`.
+
+We also have provided a metadata file, `data/rms/annotations/rms_sample_metadata.tsv`, that contains information from each sample, such as diagnosis, sex, age, etc.
+In this file each row corresponds to a sample found in the integrated SCE object.
+
+To begin, let's set up our directories and files:
+
+```{r filepaths}
 # set up file paths 
 # data directory for RMS data
 data_dir <- file.path("data", "rms")
 
-# integrated file containing samples to for DE 
+# integrated file containing all samples to use for DE analysis
 integrated_sce_file <- file.path(data_dir, "integrated", "rms_all_sce.rds")
 
-# sample metadata with information needed to setup DE
+# sample metadata with information needed to setup DE analysis
```

Comment:
These comments may be a bit long. I'm going to file an issue to check line length in the automatically rendered versions of these notebooks, as that can be an issue.
```suggestion
# sample metadata with information needed to set up DE analysis
```